lib/handler.go

```diff
@@ -95,12 +116,39 @@ func (h *handler) consume() error {
 		false,        // no-wait
 		nil,          // arguments
 	)
-	if err != nil {
-		return err
+	if err == nil {
+		h.messages = msgChan
 	}
-	for message := range messages {
-		if _, err := h.websocket.Write(message.Body); err != nil {
-			return err
+	return err
+}
+
+func (h *handler) setUpTicker() {
+	h.ticker = time.NewTicker(pingPeriod)
+	h.websocket.SetReadDeadline(time.Now().Add(pongWait))
+	h.websocket.SetPongHandler(h.receivePong)
+}
+
+func (h *handler) receivePong(string) error {
```

Comment:
It would be customary to call unused variable `_` rather than skip it's name completely, C-style.